extensions/cactus-plugin-htlc-coordinator/package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "@hyperledger/cactus-plugin-htlc-coordinator",
```

Comment:
@jordigiam Is the code in here generic enough for it to handle future HTLCs that we'll implement for other (non-EVM) ledgers as well? (think Fabric, Corda, etc.) If yes, awesome, but if not, then we'll probably need to narrow the scope in the package name to make sure that more packages can be added later on under the same naming scheme to support the other ledgers.

Reply:
We only can use the existing HTLC packages. In the future if we will have a HTLC for Fabric then the HTLC Coordinator will be use to it.

Reply:
@jagpreetsinghsasan This was resolved but I don't see the change in the diff, please go through these and double check if they are really resolved.